lib/mongoose_push/metrics/telemetry_metrics.ex

```diff
@@ -10,10 +10,13 @@ defmodule MongoosePush.Metrics.TelemetryMetrics do
   def metrics do
     [
       # Summary is not yet supported in TelemetryMetricsPrometheus
-      Telemetry.Metrics.distribution("mongoose_push.notification.send.time",
-        buckets: [100, 250, 500, 1000],
+      Telemetry.Metrics.distribution(
+        "mongoose_push.notification.send.time.microsecond",
+        event_name: [:mongoose_push, :notification, :send],
+        measurement: :time,
+        buckets: [10, 25, 50, 100, 200, 500, 1000],
```

Comment:
Those buckets seem to be for milliseconds (at least they make sense for milliseconds), while the change is for `microsecond`. Is that a mistake or am I missing something?

Reply:
When doing local testing setting the buckets in this manner resulted in having hits for some buckets. The buckets were extended to have results granularity when testing with mock and fcm.

Reply:
Do you have some intuition as to what order of magnitude those results should be for installation with some significant load?

Reply:
Those values, in `microsecond` mean that the max bucket is for 1ms. Indeed, on local environment, where mocks are on "localhost", this is possible. I don't have any actual data from production environments on that, but APNS/FCM server are always remote, so the request latency itself will be **at least** few milliseconds. I wouldn't be surprised if the whole process of sending notification would be more then 200ms, so 200x max bucket right now. That's why I said, that current buckets look like values in `milliseconds` (with last value at 1000ms == 1sec).

Reply:
Ok, we are going to adjust the buckets.